plugins/source/azure/client/multiplexer.go

```diff
@@ -35,6 +35,27 @@ func SubscriptionMultiplexRegisteredNamespace(table, namespace string) func(sche
 	}
 }
 
+func SubscriptionResourceGroupMultiplexRegisteredNamespace(table string, namespace string) func(schema.ClientMeta) []schema.ClientMeta {
+	return func(meta schema.ClientMeta) []schema.ClientMeta {
+		client := meta.(*Client)
+		var c = make([]schema.ClientMeta, 0)
+		for _, subId := range client.subscriptions {
+			if _, ok := client.registeredNamespaces[subId][namespace]; ok {
```

Comment:
Kept the logic as `SubscriptionMultiplexRegisteredNamespace` to skip non registered namespaces